docs/faq.md

```diff
@@ -18,47 +18,126 @@ nav_order: 9
+
+### What's the difference between a version and metadata history?
+
+Metadata history is a lightweight change tracking mechanism (typos, metadata corrections). Reasons for updating metadata:
+
+-   Adding a citation
+-   Minor updates to the description or the title of the item (e.g. dataset, model)
+-   Corrections of minor errors (e.g. typos, missing information) which do not constitute a significant change in the utility or purpose of the item
+
+Versioning an item creates a new entity in Provena. This includes the minting of a new identifier and new entity that is linked to the previous version of the item. It gives users the ability to signal that there is a new version of a thing. Reasons for a new version could be:
+
+No - Provena doesn't allow creating a version from a previous version, i.e.  version branching. This would raise issues such as merge conflicts. Therefore, this is currently out of scope.
+
+### Are users able to create a version from a non-current version, rather than the latest?
+
+No - Provena doesn't allow creating a version from a previous version, i.e. version branching. This would raise issues such as merging, dangling branches, etc. This feature is intentionally excluded to prioritise a simple, understandable lineage of item versions.
+
+Provena implements a simple versioning scheme where versions are created from the latest version.
+
+### What's the difference between the 'clone' feature and the 'versioning' feature?
+
+Functionality wise, the clone and versioning feature looks similar - metadata is copied, and a new identifier is minted for the item. 
+
+The difference between "clone" and "versioning" is that for a new version:
+- the new versioned item is linked to the previous version of the item by a version activity in the provenance graph
+
+### What happens when a version is created for Datasets?
+
+When a new version is created for a Dataset and if users want to use the Provena data storage feature, a new storage location is created. This storage location will be empty for users to upload relevant data for that version.
+
+Other things that happen when a version is created:
+- The new versioned dataset is linked to the previous version of the dataset
+- A version activity is generated in the provenance graph linking the versioned dataset with each other
+- Both versions of the dataset contain information in their metadata which the user can view to see the links to previous/next versions. Both are managed independently of each other
+
+### Why can't I create a version for this item in the Registry?
+
+To be able to version an item in the registry, you will need:
+- to ensure you are logged in and have Admin permissions for the item
+- to ensure you have navigated to the latest version of an item in the Registry
+- to ensure the item is an allowable type to be versioned (see below)
```

Comment:
hyperlinks here would be nice

logged in -> logging in

Admin permissions -> registry permissions